packages/atlas/src/components/NftTile/NftTileDetails.tsx

```diff
@@ -85,20 +82,25 @@ export const NftTileDetails: React.FC<NftTileDetailsProps> = ({
     },
   })
 
+  const handleCopyVideoURLClick = useCallback(() => {
```

Comment:
This should use the new hook we introduced recently, `const { copyToClipboard } = useClipboard()`